core/classes/Core/Validate.php

```diff
@@ -159,7 +159,7 @@ public static function check(array $source, array $items = []): Validate {
                         if (empty($source[$array][$matches[1]])) {
                             $missing = true;
                         }
-                    } else if (empty($value)) {
+                    } else if (empty($value) && $value !== '0') {
```

Comment:
Wasn't completely sure about this one, let me know what you think

Reply:
Only concern is that this could cause unintended/surprise behaviour elsewhere where it's used...

Reply:
Should be okay but let's keep an eye out for reports in case we are actually relying on `REQUIRED` for non-'0' values in places